### DISCUSSION TITLE
Add option for global statusbar.

### DISCUSSION THREAD

**Discussion by TornaxO7** [helix-editor/helix#8766] [Ideas] [answered]
Neovim provides something like that: A global statusline for the whole helix editor instead of a statusline for each window.

**Comment by the-mikedavis** [comment, marked as answer]
Duplicate of https://github.com/helix-editor/helix/issues/2254

**Reply by TornaxO7** [reply]
oh bruh. Sorry, I only typed `statusbar` and couldn't find anything...
